app/views/arf_reports/welcome.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("book", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Compliance Reports') %></h1>
+  <p>
+    <%= _("You don't seem to have any reports.") %></br>
```

Comment:
Looks great, could you please expand a test a bit, e.g. to

"You don't seem to have any ARF report. ARF report is a summary of a single scan occurrence on a particular host for a given Compliance Policy."